DataFormats/TrackReco/src/classes_def.xml

```diff
@@ -1,4 +1,7 @@
 <lcgdict>
+
+  <class name="edm::Wrapper<std::vector<std::pair<int,int>>>"/>
```

Comment:
@Dr15Jones 
shouldn't we define Wrapper dictionaries in StdDictionaries?
I see that there are no Wrapper dicts in StdDictionaries now.
Why?
Which logic prevents to have the same, e.g.,  ```Wrapper<std::vector<int>>``` to show up in many DF packages?